src/docs/tune-capacity.dox

```diff
@@ -0,0 +1,38 @@
+/*! @page tune_capacity Capacity tuning
+
+In some cases, it can be helpful to constrain the overall I/O bandwidth
+generated by the database. This can be beneficial when resources are shared,
+for example, in cloud or virtual environments.
+
+The total bandwidth capacity for the database is configured by setting the
+\c io_capacity configuration string when calling the ::wiredtiger_open
+function.  The capacity can be adjusted with WT_CONNECTION::reconfigure.
+
+An example of setting a capacity limit to 40MB per second:
+
+@snippet ex_all.c Configure capacity
+
+When a total capacity is set, the database ensures that the volume of
+system reads and writes totalled will not exceed the given I/O capacity.
+If a read or write is scheduled and would overflow the capacity, the issuing
+thread will sleep to guarantee the capacity ceiling.  The policy used is
+fair to all threads, and gives some weight to both readers and writers to
+try to ensure that each session can make progress when bandwidth
+resources are limited.
```

Comment:
Not sure what `the database` means in this context. I think you can just delete `the database ensures that` and not lose meaning. I'd suggest `When a total capacity is set, the total volume of system reads and writes will not exceed the given I/O capacity.`

Reply:
Fixed.